Web/Web.csproj

```diff
@@ -15,17 +15,18 @@
     </PackageReference>
     <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="9.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="5.0.9" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.DataAnnotations.Validation" Version="3.2.0-rc1.20223.4" />
```

Comment:
Is this the right package? It's an RC version released more than 1 year ago...

Reply:
Yes, sadly enough there's no official release yet, but I need it for complex form evaluations

Reply:
This is the right package, it's sadly enough an experimental package, but it does what is expected from it.